# HG changeset patch
# User Rik <rik@octave.org>
# Date 1458949072 25200
#      Fri Mar 25 16:37:52 2016 -0700
# Node ID 212bafe8413ae18b6d8efe0fadd43957aec768c2
# Parent  eb15bc7c5c0b3213a9016af17cd55d257e844509
gl2ps-print.cc: Fix compiling whet --without-opengl used.

* gl2ps-print.cc: Move #include "gl2ps-print.h" outside of HAVE_GL2PS_H ifdef.

diff --git a/libinterp/corefcn/gl2ps-print.cc b/libinterp/corefcn/gl2ps-print.cc
--- a/libinterp/corefcn/gl2ps-print.cc
+++ b/libinterp/corefcn/gl2ps-print.cc
@@ -19,31 +19,32 @@ along with Octave; see the file COPYING.
 <http://www.gnu.org/licenses/>.
 
 */
 
 #ifdef HAVE_CONFIG_H
 #  include "config.h"
 #endif
 
+// Both header files are required outside of HAVE_GLP2S_H
 #include "errwarn.h"
+#include "gl2ps-print.h"
 
 #ifdef HAVE_GL2PS_H
 
 #include <cstdio>
 #include <limits>
 #include <unistd.h>
 
 #include <gl2ps.h>
 
 #include "lo-mappers.h"
 #include "oct-locbuf.h"
 #include "unwind-prot.h"
 
-#include "gl2ps-print.h"
 #include "gl-render.h"
 #include "oct-opengl.h"
 #include "sighandlers.h"
 #include "sysdep.h"
 #include "text-renderer.h"
 
 class
 OCTINTERP_API
@@ -818,13 +819,11 @@ gl2ps_print (const graphics_object& fig,
   gl2ps_renderer rend (fp, term);
 
   rend.draw (fig, stream);
 
   // Make sure buffered commands are finished!!!
   rend.finish ();
 
 #else
-
   err_disabled_feature ("gl2ps_print", "gl2ps");
-
 #endif
 }
